driver/remote/driver_windows.go

```diff
@@ -0,0 +1,46 @@
+//go:build windows
+// +build windows
```

Comment:
Also, don't think we need these build constraints (though we are inconsistent with it) - the filename should be the indicator in this case.